testgrid/config/config.yaml

```diff
@@ -3648,6 +3648,18 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gci-gce-alpha-features
     base_options: 'include-filter-by-regex=Volume%7Cstorage'
     description: 'storage gci gce alpha features e2e tests for master branch'
+  - name: gci-gke-multizone
+    test_group_name: ci-kubernetes-e2e-gci-gke-multizone
+    base_options: 'include-filter-by-regex=Volume%7Cstorage'
+    description: 'storage gci gke multizone e2e tests for master branch'
+  - name: gke-multizone
+    test_group_name: ci-kubernetes-e2e-gke-multizone
```

Comment:
Is this different from the gci version?  Isn't everything on gci now?

Reply:
yep.. they are running the same thing now, seems the multizone job is slipped from the cleanup, I'll delete one of them.

Reply:
cc @krzyzacy ^
I think so?